multi-int-store/src/test/resources/application.yml

```diff
@@ -3,4 +3,5 @@ endpointUrl:
 aws:
   s3:
     bucket:
-      quarantine: quarantine
\ No newline at end of file
+      quarantine: quarantine
+    endpointUrl: http://localhost:1234/s3
```

Comment:
❓Why is `aws.s3.endpointUrl` needed but not `aws.s3.region`?

Reply:
I'm actually surprised that the App Context was able to be configured without this. For clarification, you typically only include the the endpointUrl if you want to override the location of S3. (For example, the Mino S3 that we use for testing)

Reply:
Both of those values are in the same constructor. I'm confused why you only need one.

Reply:
Actually we don't need either, I forgot that we mock the S3Client in the tests. We need the bucket name for property injection when the S3Adaptor bean gets created.